controllers/vaultstaticsecret_controller.go

```diff
@@ -218,11 +245,242 @@ func (r *VaultStaticSecretReconciler) handleDeletion(ctx context.Context, o clie
 	return nil
 }
 
+func (r *VaultStaticSecretReconciler) ensureEventWatcher(ctx context.Context, o *secretsv1beta1.VaultStaticSecret, c vault.Client) error {
+	logger := log.FromContext(ctx)
```

Comment:
Could we also set the loggers name e.g: ` logger := log.FromContext(ctx).WithName("ensureEventWatcher")`